src/views/dashboard/User.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Query } from "react-apollo";
+import { GET_USERS } from "../../graphql-requests/queries";
+import "../home/Home.css";
+
+const user_id = localStorage.getItem("user_id");
+
+const User = props => (
+   <Query query={GET_USERS}>
+      {({ loading, data }) => {
+         if (loading) return <p>Loading...</p>;
+         return (
+            <>
+               {/*console.log(data)*/}
+               <div>
+                  {data.users.map(user => (
+                     <div key={user.id} value={user.username}>
+                        {user.id === `${user_id}` ? (
+                           <>
```

Comment:
I'm not sure if we need both a fragment and a div here. We should be okay to just pick one or the other.

Reply:
For some reason the display only works well with both in this case, and as it gets refactored to typescript we can address that which I anticipate to be hard (for me at least :)